libs/ts-rest/core/src/lib/client.ts

```diff
@@ -108,16 +108,14 @@ export const defaultApi: ApiFetcher = async ({
   const result = await fetch(path, { method, headers, body, credentials });
   const contentType = result.headers.get('content-type');
 
-  if (!contentType) {
-    throw TypeError('Resource Response missing content-type header');
-  }
-
-  if (contentType.includes('application/json')) {
-    return { status: result.status, body: await result.json() };
-  }
+  if (contentType) {
+    if (contentType.includes('application/json')) {
+      return { status: result.status, body: await result.json() };
+    }
 
-  if (contentType.includes('text/plain')) {
-    return { status: result.status, body: await result.text() };
+    if (contentType.includes('text/plain')) {
```

Comment:
Is "contentType?." Nicer than the double nested if? 🤔

Reply:
Absolutely! Still stuck in ES5 lol 🤦‍♂️